### PR TITLE
R gdtools

### PR DESCRIPTION
**Pull request by funnell**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

New contributor, please review.

### CODE REVIEW

File: recipes/r-gdtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  run:
+    - r-base
+    - r-rcpp
+    - r-withr
+    - xorg-libxrender
+    - xorg-libxext
+    - xorg-libxdmcp
+    - xorg-libxau
+    - cairo [osx]
```
Comment (bgruening) [inline recipes/r-gdtools/meta.yaml]: why cairo only on OSX?

File: recipes/r-gdtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    - lib/R/lib/
+    - lib/
+
+# Suggests: htmltools, testthat, fontquiver (>= 0.2.0)
+requirements:
+  build:
+    - r-base
+    - r-rcpp
+    - r-withr
+    - posix                # [win]
```
Comment (bgruening) [inline recipes/r-gdtools/meta.yaml]: posix and the native-toolchain is not needed

File: recipes/r-gdtools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    - xorg-libxext
+    - xorg-libxdmcp
+    - xorg-libxau
+    - cairo [osx]
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('gdtools')"  # [not win]
+    - "\"%R%\" -e \"library('gdtools')\""  # [win]
```
Comment (bgruening) [inline recipes/r-gdtools/meta.yaml]: win specific lines can also be removed

File: recipes/r-gdtools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  license: GPL-3 | file LICENSE
+  summary: Useful tools for writing vector graphics devices.
+  license_family: GPL3
+
+# The original CRAN metadata for this package was:
+
+# Package: gdtools
+# Version: 0.1.4
+# License: GPL-3 | file LICENSE
+# Title: Utilities for Graphical Rendering
```
Comment (bgruening) [inline recipes/r-gdtools/meta.yaml]: Can you clean this up and remove not useful comments. Thanks!

File: recipes/r-gdtools/meta.yaml (hunk 1 above)
Comment (funnell) [inline recipes/r-gdtools/meta.yaml]: Travis builds seemed to work on linux without it. I only added cairo so that the macOS builds would finish.

Comment (bgruening) [inline recipes/r-gdtools/meta.yaml]: Mh, but does the tool needs cairo? I suppose for drawing under some environments?

Comment (funnell) [inline recipes/r-gdtools/meta.yaml]: Looking at this issue (https://github.com/davidgohel/gdtools/issues/24), it seems that cairo is needed on macOS.

**Comment by bgruening** [comment]
Thanks @funnell!

**Pull request merged**